src/PrestaShopBundle/Controller/Admin/Sell/Order/InvoicesController.php

```diff
@@ -121,6 +132,138 @@ public function processAction(Request $request)
         return $this->redirectToRoute('admin_order_invoices');
     }
 
+    /**
+     * @return array
+     *
+     * @throws FieldNotFoundException
+     *
+     * @var InvalidConfigurationDataErrorCollection
+     */
+    private function getErrorMessages(InvalidConfigurationDataErrorCollection $errors): array
+    {
+        $messages = [];
+
+        foreach ($errors as $error) {
+            $messages[] = $this->getErrorMessage($error);
+        }
+
+        return $messages;
+    }
+
+    /**
+     * @param InvalidConfigurationDataError $error
+     *
+     * @return string
+     *
+     * @throws FieldNotFoundException
+     */
+    private function getErrorMessage(InvalidConfigurationDataError $error): string
+    {
+        switch ($error->getErrorCode()) {
+            case InvoicesByDateDataProvider::ERROR_INVALID_DATE_TO:
+            case InvoicesByDateDataProvider::ERROR_INVALID_DATE_FROM:
+            return $this->trans(
+                    'Invalid "%s" date.',
```

Comment:
I don't understand what the variable will be replaced with? Could you provide an example, please? Thanks in advance!

Reply:
So before we had ```Invalid "From" date.``` and ```Invalid "To" date.``` error messages, this is basically the same thing but variable is label of the field. 
![image](https://user-images.githubusercontent.com/47781757/111492542-6dd37580-8745-11eb-9227-a49f01511361.png)
I figured it's the same message so I would join them into one. But it's not a problem to have two messages (like it was before) without variable if that's preferable.